src/treq/test/test_client.py

```diff
@@ -49,6 +49,18 @@ def test_request_uri_idn(self):
             b'GET', b'http://xn--bea.net',
             Headers({b'accept-encoding': [b'gzip']}), None)
 
+    def test_request_uri_idn_params(self):
+        """
+        A URL that contains non-ASCII characters can be augmented with
+        querystring parameters.
+
+        This reproduces treq #264.
```

Comment:
Would you mind including the github link, rather than the unqualified ticket number? one day we'll all move back to sourceforge and then we'll have to renumber everything